frontend/src/views/Layout/ContentFooter.vue

```diff
@@ -39,6 +39,17 @@
           >
             {{ $t('whitepaper') }}
           </b-nav-item>
+          <b-nav-item
+            :href="
+              $i18n.locale === 'de'
+                ? 'https://gradido.net/de/contact/'
+                : 'https://gradido.net/en/contact/'
```

Comment:
This does not support more then 2 languages - we should refactor this. This link should be part of the language file.

Reply:
geändert